cmd/veneur-prometheus/main.go

```diff
@@ -6,20 +6,24 @@ import (
 	"io"
 	"math"
 	"net/http"
+	"strings"
 	"time"
 
 	"github.com/DataDog/datadog-go/statsd"
 	dto "github.com/prometheus/client_model/go"
 	"github.com/prometheus/common/expfmt"
+	glob "github.com/ryanuber/go-glob"
 	"github.com/sirupsen/logrus"
 )
 
 var (
-	debug       = flag.Bool("d", false, "Enable debug mode")
-	metricsHost = flag.String("h", "http://localhost:9090/metrics", "The full URL — like 'http://localhost:9090/metrics' to query for Prometheus metrics.")
-	interval    = flag.String("i", "10s", "The interval at which to query. Value must be parseable by time.ParseDuration (https://golang.org/pkg/time/#ParseDuration).")
-	prefix      = flag.String("p", "", "A prefix to append to any metrics emitted. Do not include a trailing period.")
-	statsHost   = flag.String("s", "127.0.0.1:8126", "The host and port — like '127.0.0.1:8126' — to send our metrics to.")
+	debug             = flag.Bool("d", false, "Enable debug mode")
+	metricsHost       = flag.String("h", "http://localhost:9090/metrics", "The full URL — like 'http://localhost:9090/metrics' to query for Prometheus metrics.")
+	interval          = flag.String("i", "10s", "The interval at which to query. Value must be parseable by time.ParseDuration (https://golang.org/pkg/time/#ParseDuration).")
+	blockedLabelsStr  = flag.String("l", "", "A comma-seperated list of label name globs to not export")
```

Comment:
Can we use --long-options here? `-l` and `-m`, to me, don't immediately make their purpose clear like `--ignore-labels-matching` and `--ignore-metrics-matching` would.

Reply:
Go's flag package doesn't use GNU-style options, so you can do `-foo` but not `--foo`.

Reply:
Sure, that's fine. It's not the number of dashes that worries me, but the short char vs long name.